pkg/route/api/types.go

```diff
@@ -101,4 +109,11 @@ const (
 	TLSTerminationPassthrough TLSTerminationType = "passthrough"
 	// TLSTerminationReencrypt terminate encryption at the edge router and re-encrypt it with a new certificate supplied by the destination
 	TLSTerminationReencrypt TLSTerminationType = "reencrypt"
+
+	// TLSInsecureDisable disables insecure connections for an edge-terminated route.
+	TLSInsecureDisable TLSInsecureType = "disable"
```

Comment:
All constants should be `CamelCase` - don't change the old ones, but any new ones should follow that rule.